tree/tree/src/TTree.cxx

```diff
@@ -4384,13 +4384,24 @@ Int_t TTree::Fill()
    if (fBranchRef) {
       fBranchRef->Clear();
    }
+
+   std::unique_ptr<TBranchIMTHelper> imt_helper;
```

Comment:
humm ... this made me take another look at the code and ... I am getting concerned on the sheer amount of #ifdef IMT that we are ending with ... [For this particular case, we could also have an empty TBranchIMTHelper  when IMT is disable and add a  bool  on/off to test against (rather than nullptr).